src/GitPrompt.ps1

```diff
@@ -112,10 +113,6 @@ $global:GitPromptSettings = [pscustomobject]@{
     TruncatedBranchSuffix                       = '...'
 }
 
-$isAdminProcess = Test-Administrator
```

Comment:
This value can't change in the context of a host instance, so let's keep this cached here.